### PR TITLE
Create ruleset: ExtremeTech.com

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/ExtremeTech.com.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	<target host="product4.extremetech.com" />
+	<target host="static.extremetech.com" />
+
+	<securecookie host="^www\.extremetech\.com$" name=".+" />
+
+	<rule from="^http://extremetech\.com/"
+		  to="https://www.extremetech.com/" />
+
+	<rule from="^http://metrics\.extremetech\.com/"
+		  to="https://extremetech-com.122.2o7.net/" />
```
Comment (J0WI) [inline src/chrome/content/rules/ExtremeTech.com.xml]: Please add a test url for it.

Comment (injust) [reply src/chrome/content/rules/ExtremeTech.com.xml]: I can't find any requests for `metrics.extremetech.com`, so I can't add tests. Removing.

Comment (J0WI) [reply src/chrome/content/rules/ExtremeTech.com.xml]: See e.g. `http://metrics.extremetech.com/1x1.gif`

Comment (injust) [reply src/chrome/content/rules/ExtremeTech.com.xml]: Adding that, thanks.

Comment (injust) [reply src/chrome/content/rules/ExtremeTech.com.xml]: That actually 404s on `extremetech-com.122.2o7.net`, so removing the target entirely.

File: src/chrome/content/rules/ExtremeTech.com.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!--
+	Invalid certificate:
+		- discuss.extremetech.com
+		- metrics.extremetech.com, equivalent to extremetech-com.122.2o7.net
```
Comment (J0WI) [inline src/chrome/content/rules/ExtremeTech.com.xml]: Is this comment still true?

Comment (injust) [reply src/chrome/content/rules/ExtremeTech.com.xml]: ...not sure how I missed that. Removing now.

**Pull request merged**